ck.sh

```diff
@@ -85,6 +85,53 @@ case "$COMMAND" in
 		esac
 		;;
 
+	buildskin)
+		cd skins
+
+		echo "Which skin do you want to build?"
+
+		select skin in *
+		do
+			echo "you selected $skin"
+			
+			cd ../ckeditor-dev
+
+			java -jar dev/builder/ckbuilder/2.3.2/ckbuilder.jar --build-skin ../skins/$skin ../ckeditor/skins/$skin
+
+			break
+		done
+		;;
+
+	createskin)
+		read -r -p "What is the name of the new skin? " skinName
+
+		echo "Which skin do you want to use as base?"
```

Comment:
Do we need to offer this much flexibility? Won't we be building exactly one skin from one base? If this is not something that we are going to do often, then we might be able to simplify it.

Reply:
Probably we don't need it, I need to keep working on not doing more than we need 😅 I'll get rid of it.

Reply:
To help you, you can print these out and sleep with them under your pillow:

- https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it
- https://martinfowler.com/bliki/Yagni.html
- https://wiki.c2.com/?YouArentGonnaNeedIt

Alternatively, you could read them. Another option is starting on wiki.c2.com and spending the next 10 hours following links. 😂